docs/shared/using-nx/affected.md

```diff
@@ -1,8 +1,13 @@
 # Run Only Tasks Affected by a PR
 
-As your workspace grows, re-testing, re-building and re-linting all projects becomes too slow. To address this, Nx is able to determine the minimum set of projects that were affected by the change, and **only run tasks on those projects**. This drastically improves the speed of your CI and the amount of compute that needs to be run, even before other features like [remote caching](/ci/features/remote-cache) and [distributed task execution](/ci/features/distribute-task-execution) are taken into account.
+As your workspace grows, re-testing, re-building, and re-linting **all projects becomes too slow**. To address this, Nx comes with a so-called "affected" command. Using this command, Nx
```

Comment:
Why are we saying `so-called "affected" command`? It's literally an `affected` command...